src/app/products/productManagement/product-add/product-add.component.ts

```diff
@@ -0,0 +1,47 @@
+import { ProductService } from './../../../services/product.service';
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-product-add',
+  templateUrl: './product-add.component.html',
+  styleUrls: ['./product-add.component.scss']
+})
+export class ProductAddComponent implements OnInit {
+  
+  quantityLabel : String = "Quantity";
```

Comment:
change these fields to reflect what the backend receives

Reply:
what may work even better is to instead make a Product and a Taxonomy instance here in the class and access the fields for that class